### ISSUE TITLE
Search for all building within a bounding box 

### ISSUE THREAD

**Issue by kwandimi** [georocket/georocket#170]
Hello Guys am currently working on a project with the open version og Georocket and need to request using the spatial querry all buildings with a bbox. can anyone please help with that ? i have tried to follow the doc but no success.

**Comment by michel-kraemer** [comment]
Hi @kwandimi! Can you please elaborate what you tried and what exactly did not work? What kind of source data do you have? Have you imported and indexed the data with GeoRocket? Do you get an error message? Or do you get an empty response? Did you check your spatial reference system? Etc. Thanks.

**Comment by kwandimi** [comment]
Hello, @michel-kraemer  thanks for your response. I uploaded a gml file on my georocket server as explained in the documentation. currently, the search by gml ID works fine by responding with a gml file containing the appropriate gml building ID. however, when I make a spatial query it does not respond with the building with the spatial bbox. Usually, when I make a spatial query I should get a gml file containing building within the spatial query but in this case, I got only the "gml: Envelope" of these buildings as shown in the attached image. 

once again thanks and look forward to your response.

<img width="919" alt="Capture" src="https://user-images.githubusercontent.com/43406649/97162193-242ac400-177f-11eb-85a6-73ac576cf6c0.PNG">
